Source/Core/Common/Debug/Watches.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/Debug/Watches.h"
+
+#include <algorithm>
+#include <sstream>
+
+namespace DebugCommon
```

Comment:
This is better suited as being in the Common namespace and being within a Debug namespace inside it. This way it's still under the common namespace like everything else in common (anything that isn't following that pattern should be)